hal/src/electron/modem/mdm_hal.cpp

```diff
@@ -1267,6 +1267,10 @@ bool MDMParser::powerState(void) const {
     state = HAL_GPIO_Read(RI_UC);
     // If RI is still low - we are off
     // If RI is high - we are on
+    if (getModemStateChangeCount() == 0) {
+        _incModemStateChangeCount();
+        _pwr = state;
+    }
```

Comment:
@avtolstoy Would you like to add this change to the PR: https://github.com/particle-iot/device-os/pull/2217 as well? This could reduce 1 s that is cost on verifying the power state on electron.